src/core/client/framework/lib/errors/relayNetworkRequestError.ts

```diff
@@ -0,0 +1,28 @@
+import { RRNLRequestError } from "react-relay-network-modern/es";
+
+/**
+ * RelayNetworkRequestError wraps Request errors thrown by Relay Network Layer.
+ */
+export default class RelayNetworkRequestError extends Error {
+  // Keep origin of original server response.
+  public origin: RRNLRequestError;
+
+  constructor(error: RRNLRequestError) {
+    let msg = "An unexpected network error occured, please try again later.";
+    if (error.res) {
+      msg += ` [Code: ${error.res.status.toString()}`;
+      if (error.res.statusText) {
+        msg += " " + error.res.statusText;
```

Comment:
Lol, I like how we use two different string manipulations to create a space here. Not a big deal, but it made me smile